File: backend/DocCareWeb/src/DocCareWeb.Domain/Entities/Doctor.cs
namespace DocCareWeb.Domain.Entities
{
    public class Doctor : BaseEntity
    {
        public string Name { get; set; } = string.Empty;
        public int SpecialtyId { get; set; }
        public Specialty Specialty { get; set; } = null!;
        public string? Crm { get; set; }
        public string? Email { get; set; }
        public string? CellPhone { get; set; }
        public string? Phone { get; set; }
    }
}

File: frontend/doccareweb/src/app/features/doctor/models/doctor.ts
import { Specialty } from "./specialty";

export interface DoctorBase {
    name: string;
    crm?: string;
    email?: string;
    cellPhone?: string;
    phone?: string;
  }

  export interface Doctor extends DoctorBase {
    id: number; 
    specialtyId: number; 
  }

  export interface DoctorList extends DoctorBase {
    id: number; 
    specialty: Specialty; 
  }